### ISSUE TITLE
flang: make install fail to install flang/ISO_Fortran_binding.h

### ISSUE THREAD

**Issue by sylvestre** [llvm/llvm-project#69697]
started to fail yesterday:

```

-- Installing: /build/source/debian/tmp/usr/lib/llvm-18/include/flang/Optimizer/Dialect/FIROps.h.inc
-- Installing: /build/source/debian/tmp/usr/lib/llvm-18/include/flang/CMakeFiles
CMake Error at tools/flang/cmake_install.cmake:107 (file):
  file INSTALL cannot find
  "/usr/lib/llvm-18/include/flang/ISO_Fortran_binding.h": No such file or directory.
```

**Comment by sylvestre** [comment]
@psteinfeld i guess this is the regressor?
https://github.com/llvm/llvm-project/commit/e6e90840708099425b7b69dd053634ff25d4907f

**Comment by psteinfeld** [comment]
> @psteinfeld i guess this is the regressor? [e6e9084](https://github.com/llvm/llvm-project/commit/e6e90840708099425b7b69dd053634ff25d4907f)

Yes.  @jeanPerier fixed things up for now with https://github.com/llvm/llvm-project/commit/54881c9225e612830a9e78574a805309b516627b

**Comment by jeanPerier** [comment]
> Yes. @jeanPerier fixed things up for now with [54881c9](https://github.com/llvm/llvm-project/commit/54881c9225e612830a9e78574a805309b516627b)

No, my commit removed _the test only_ because it failed with gcc. The issue here is different, this is a CMAKE error that is caused by the CMake change that I did not revert (I did not notice a build bot failing with this CMake error).

@sylvestre, could you share your cmake config/what compilers you are using to help reproducing the issue?

**Comment by sylvestre** [comment]
Sure here is the full log
https://llvm-jenkins.debian.net/job/llvm-toolchain-bionic-binaries/architecture=amd64,distribution=bionic,label=amd64/1365/consoleFull

(I am on my phone. This is why it is hard to give you the exact command)
